### PR TITLE
Migration Java SDK to Asgradio

### PR DESCRIPTION
**Pull request by gayashanbc**

### CODE REVIEW

File: components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/AESDecryptor.java [hunk 1]
```diff
@@ -16,13 +16,13 @@
  * under the License.
  */
 
-package org.wso2.carbon.identity.sso.agent.saml;
+package org.wso2.carbon.identity.sso.agent.sdk;
```
Comment (darshanasbg) [inline components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/AESDecryptor.java]: Rename to io.asgardio.java.saml.sdk?

File: components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/SAML2SSOManager.java [hunk 2]
```diff
@@ -88,18 +88,18 @@
...
-import org.wso2.carbon.identity.sso.agent.saml.exception.ArtifactResolutionException;
-import org.wso2.carbon.identity.sso.agent.saml.exception.InvalidSessionException;
-import org.wso2.carbon.identity.sso.agent.saml.exception.SSOAgentException;
-import org.wso2.carbon.identity.sso.agent.saml.internal.SSOAgentServiceComponent;
-import org.wso2.carbon.identity.sso.agent.saml.security.X509CredentialImpl;
-import org.wso2.carbon.identity.sso.agent.saml.session.management.SSOAgentSessionManager;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentConstants;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentDataHolder;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentUtils;
+import org.wso2.carbon.identity.sso.agent.sdk.artifact.SAMLSSOArtifactResolutionService;
```
Comment (darshanasbg) [inline components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/SAML2SSOManager.java]: Move to io.asgardio.java.saml.sdk

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml [hunk 3]
```diff
@@ -19,23 +19,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <parent>
-        <groupId>org.wso2.carbon.identity.agent.sso.java</groupId>
-        <artifactId>identity-agent-sso</artifactId>
+        <groupId>io.asgardio.java.saml.sdk</groupId>
+        <artifactId>java-saml-sdk</artifactId>
         <relativePath>../../pom.xml</relativePath>
         <version>5.5.9-SNAPSHOT</version>
```
Comment (darshanasbg) [inline features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Update versions in a separate commit

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml [hunk 4]
```diff
@@ -19,23 +19,23 @@
...
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.carbon.identity.sso.agent.server.feature</artifactId>
+    <artifactId>io.asgardio.java.saml.sdk.server.feature</artifactId>
     <packaging>pom</packaging>
-    <name>SSO Agent server feature</name>
-    <url>http://wso2.org</url>
-    <description>This feature contains sso agent which is a library for sso and openid</description>
+    <name>Java SAML SDK server feature</name>
```
Comment (darshanasbg) [inline features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Use same naming format. ie. Lets use title case for <name> in every pom.
"Java SAML SDK Server Feature"

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml [hunk 5]
```diff
@@ -89,7 +89,7 @@
                                 </properties>
                             </adviceFile>
                             <bundles>
-                                <bundleDef>org.wso2.carbon.identity.agent.sso.java:org.wso2.carbon.identity.sso.agent</bundleDef>
+                                <bundleDef>io.asgardio.java.saml.sdk:io.asgardio.java.saml.sdk</bundleDef>
                                 <bundleDef>org.wso2.orbit.org.opensaml:opensaml</bundleDef>
                                 <bundleDef>org.wso2.orbit.joda-time:joda-time</bundleDef>
                                 <bundleDef>org.wso2.orbit.org.openid4java:openid4java</bundleDef>
```
Comment (darshanasbg) [inline features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Do we need openid4java?

**Review by darshanasbg — CHANGES_REQUESTED** [review]

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml (hunk 5 above)
Comment (gayashanbc) [reply features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Removed in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml (hunk 4 above)
Comment (gayashanbc) [reply features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e

File: components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/SAML2SSOManager.java (hunk 2 above)
Comment (gayashanbc) [reply components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/SAML2SSOManager.java]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e

File: components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/AESDecryptor.java (hunk 1 above)
Comment (gayashanbc) [reply components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/AESDecryptor.java]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e

File: features/io.asgardio.java.saml.sdk.server.feature/pom.xml (hunk 3 above)
Comment (gayashanbc) [reply features/io.asgardio.java.saml.sdk.server.feature/pom.xml]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/0785a407e47099951350f5e95fecd0f16e4d6c13

**Comment by gayashanbc** [comment]
Closing this PR as the review process completed

**Pull request closed without merging**